### PR TITLE
Sections

### PR DESCRIPTION
**Pull request by narthur**

### CODE REVIEW

File: src/components/organisms/testimonies.module.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+.body {
+	font-size: 1.25rem;
+	color: black;
+	margin-top: 0;
+}
+
+.author {
+	// TODO: Update with specific serif font; DRY
+	font-family: serif;
```
Comment (mattleff) [inline src/components/organisms/testimonies.module.scss:17]: I think we should use a variable now that we can update once we figure out what the ultimate serif font will be.

File: src/lib/test/helpers.tsx [hunk 2]
```diff
@@ -87,13 +87,18 @@ export function buildRenderer<
...
 ): Renderer<P> {
-	const { getProps = undefined, defaultParams = {} } = options;
+	const {
+		getProps = undefined,
+		defaultParams = {},
+		defaultProps = {},
+	} = options;
 	return async (options: RendererOptions<P> = {}): Promise<RenderResult> => {
-		const { params = {}, props = {}, router = {} } = options;
+		const { params = {}, props, router = {} } = options;
```
Comment (mattleff) [inline src/lib/test/helpers.tsx:99]: I'm sure there's a reason for all this, but it's starting to feel like there's an awful lot of ways (code branches) to get the `props_`... If all these arguments are necessary, maybe the default value of `getProps` could be a closure that returns `options.defaultProps`?

File: src/lib/test/helpers.tsx [hunk 3]
```diff
@@ -87,13 +87,18 @@ export function buildRenderer<
...
 		const p = { ...defaultParams, ...params };
```
Comment (mattleff) [inline src/lib/test/helpers.tsx:100]: Also, maybe `p` could be `fullParams` or something along those lines?

File: src/components/organisms/slider.tsx [hunk 4]
```diff
@@ -25,7 +31,15 @@ export default function CardSlider({ children }: CardSliderProps): JSX.Element {
...
+			style={
+				{
+					'--perSlide': perSlide,
+					overflow: clip ? 'hidden' : 'visible',
+				} as CSSProperties
+			}
+		>
 			<div
 				data-testid={'card-window'}
 				className={styles.cards}
```
Comment (mattleff) [inline src/components/organisms/slider.tsx:45]: `styles.slides`?

File: src/components/organisms/slider.tsx [hunk 5]
```diff
@@ -1,21 +1,27 @@
...
 // TODO: improve children type
-interface CardSliderProps {
+interface SliderProps {
 	children: ReactNode;
+	perSlide?: number;
+	clip?: boolean;
 }
 
-export default function CardSlider({ children }: CardSliderProps): JSX.Element {
+export default function Slider({
```
Comment (mattleff) [inline src/components/organisms/slider.tsx:16]: Good clarifying rename 👍🏻

**Review by mattleff — APPROVED** [review]
Looks really good! 🚢

File: src/lib/test/helpers.tsx (hunk 2 above)
Comment (narthur) [reply src/lib/test/helpers.tsx:99]: Added todo, punting.

**Pull request merged**